IPython/nbconvert/utils/lexers.py

```diff
@@ -49,10 +49,14 @@
 line_re = re.compile('.*?\n')
 
 ipython_tokens = [
-  (r'(\%+)(\w+)\s+(\.*)(\n)', bygroups(Operator, Keyword,
+  (r'(%)(sx|sc|system)(.*)(\n)', bygroups(Operator, Keyword,
                                        using(BashLexer), Text)),
-  (r'(\%+)(\w+)\b', bygroups(Operator, Keyword)),
```

Comment:
I believe cell magics are handled at a higher level in [filters](https://github.com/ipython/ipython/blob/3.0.0-dev/IPython/nbconvert/filters/highlight.py#L118) and we don't get to this code if the cell starts with `%%`, but I don't see any harm in being defensive here.  

The original regexp is plain wrong.  The backslash is redundant and it matches any number of `%`'s.  What would you say about `(%%?)`?